src/Symfony/Installer/DownloadCommand.php

```diff
@@ -221,7 +222,14 @@ protected function getGuzzleClient()
             $defaults['debug'] = true;
         }
 
-        return new Client(array('defaults' => $defaults));
+        try {
+            $handler = Utils::getDefaultHandler();
+        } catch (\RuntimeException $e) {
+            throw new \RuntimeException(
+                'The Symfony installer requires the php-curl extension or the allow_url_fopen ini setting.');
```

Comment:
`);` should be on the next line, or keep everything inlined with the `\RuntimeException` constructor call.